### PR TITLE
host: Change Matrix password requirements

### PR DESCRIPTION
**Pull request by backspace**
This changes to only requiring eight characters, as [NIST has observed](https://pages.nist.gov/800-63-3/sp800-63b.html#a3-complexity) that symbol and number requirements result in predictable use of `!` and `1`, which doesn’t increase security.

I also removed a test that appeared redundant to me.

### CODE REVIEW

File: packages/matrix/tests/registration-with-token.spec.ts [hunk 1]
```diff
@@ -288,92 +288,8 @@ test.describe('User Registration w/ Token', () => {
...
-        '[data-test-password-field] ~ [data-test-boxel-input-error-message]',
-      ),
-      'password error message does not appear',
-    ).toHaveCount(0);
-
-    await page.locator('[data-test-register-btn]').click();
-    await expect(page.locator('[data-test-token-field]')).toHaveCount(1);
-  });
-
-  test(`it shows an error when password doesn't meet the requirement`, async ({
```
Comment (backspace) [inline packages/matrix/tests/registration-with-token.spec.ts:334]: This test appeared to me to be a duplicate of the above? But maybe I missed something 🤔

Comment (IanCal) [reply packages/matrix/tests/registration-with-token.spec.ts:334]: looks identical to me apart from the test name, copying and searching for the entire body matches.

**Review by IanCal — APPROVED** [review]
Meets the req in the ticket, looks good.

NIST does recommend having other checks though, like filtering on dictionary words - currently "password" is a validly complex password for us.

**Review by habdelra — APPROVED** [review]

**Comment by backspace** [comment]
> NIST does recommend having other checks though, like filtering on dictionary words - currently "password" is a validly complex password for us.

indeed, I was originally going to use that to drive the failing tests and then realised it wouldn’t fail 😬 I did a brief search on npm packages that would support this and they either seem abandoned or overly complex. But I think this is something we should improve before any wider release, thanks!

**Pull request merged**